nemo_skills/dataset/ojbench/prepare.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
does this require data_dir usage? If so, please mention this in the docs and give example command